src/notify.c

```diff
@@ -42,7 +44,7 @@
 #define FANOTIFY_BUFFER_SIZE 8192
 
 // External variables
-extern volatile int stop;
+extern atomic_int stop;
```

Comment:
This seems to be dropping the volatile keyword where its preserved in other places.